sqlite_query.c

```diff
@@ -14,26 +14,34 @@
 #include "sqlite_fdw.h"
 
 #include <stdio.h>
+#include <assert.h>
```

Comment:
Could you explain the usage of this header?

Reply:
No, I can not explain. This header was deleted. Maybe this was draft artifact from unsuccessful branch.

Reply:
Thank for updating.

Reply:
Resolved.